helm/cas-airflow/values.yaml

```diff
@@ -1,15 +1,6 @@
 route:
   clusterDomain: apps.silver.devops.gov.bc.ca
 
-gcpLogsUpload:
```

Comment:
Is this now being taken care of in another place & PR? Or is it simply no longer needed. (We talked today about the GCP logs from DAG runs being important to store in GCP, but I am not sure that those are the logs this is referencing)

Reply:
These values were configuring the old bucket provisioning job - I couldn't find another use for them.
The bucket configuration for the job upload is in `.Values.airflow.config.logging.remote_base_log_folder`
The terraform provisioning has been moved to @joshgamache 's chart